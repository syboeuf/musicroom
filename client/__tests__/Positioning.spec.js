const MultiLineString = require("../dataIn/MultiLineString");

describe("MultiLineString", () => {
  test("MultiLineString test", () => {
    const expected = "Hello\nWorld\nHello\nWorld\nHello\nWorld\n";
    expect(
      MultiLineString.join("Hello", "World", "Hello", "World", "Hello", "World")
    ).toEqual(expected);
  });
  test("MultiLineString snapshot test", () => {
    expect(
      MultiLineString.join("Hello", "World", "Hello", "World", "Hello", "World")
    ).toMatchSnapshot();
  });
});
