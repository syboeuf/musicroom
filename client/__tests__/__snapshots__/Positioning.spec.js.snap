// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiLineString MultiLineString snapshot test 1`] = `
"Hello
World
Hello
World
Hello
World
"
`;
